lib/index.d.ts

```diff
@@ -0,0 +1,124 @@
+import type { Boom } from "@hapi/boom";
+import type { AssertionError } from "assert";
+
+export type BounceErrorType = Error | "system" | "boom" | Record<any, any>;
+
+export interface BounceOptions {
+  /**
+   * An object which is assigned to the `err`, copying the properties onto the error.
+   */
+  decorate?: Record<any, any>;
```

Comment:
Again, `Record<any, any>` seems to be used wrong. I would do `{ [key: string]: any }`.